lessons/operator/03/README.md

```diff
@@ -1,370 +1,212 @@
-# Lesson 3: Introduction to Sensu Go
-
-- [Web App](#web-app)
-  - [Dashboard](#dashboard)
-  - [Namespaces switcher](#namespaces-switcher)
-  - [Event list & event detail views](#event-list--event-detail-views)
-  - [Entity list & entity detail views](#entity-list--entity-detail-views)
-  - [Silencing](#silencing)
-  - [Configuration](#configuration)
-- [EXERCISE 1: log in to the Sensu web app](#exercise-1-log-in-to-the-sensu-web-app)
-- [Command Line Interface (CLI)](#command-line-interface--cli)
-  - [API client](#api-client)
-  - [Configuration management](#configuration-management)
-  - [Inventory](#inventory)
-  - [Output formats](#output-formats)
-  - [Interactive and non-interactive modes](#interactive-and-non-interactive-modes)
-- [EXERCISE 2: install and configure `sensuctl`](#exercise-2-install-and-configure-sensuctl)
-- [EXERCISE 3: use the `sensuctl --help` command](#exercise-3-use-the-sensuctl---help-command)
-- [EXERCISE 4: inspect an event in JSON format](#exercise-4-inspect-an-event-in-json-format)
-- [EXERCISE 5: explore Sensu resources using `sensuctl`](#exercise-5-explore-sensu-resources-using-sensuctl)
-- [EXERCISE 6: create an API Key for personal use](#exercise-6-create-an-api-key-for-personal-use)
-- [EXERCISE 7: export Sensu resources using `sensuctl dump`](#exercise-7-export-sensu-resources-using-sensuctl-dump)
+# Lesson 3: Using the Sensu CLI
+- [Goals](#goals)
+- [The `sensuctl` Command Line Interface (CLI)](#the-sensuctl-command-line-interface--cli)
+   - [EXERCISE 1: Install the `sensuctl` CLI](#exercise-1-install-the-sensuctl-cli)
+- [Configuration Options](#configuration-options)
+   - [EXERCISE 2: Configure the `sensuctl` CLI](#exercise-2-configure-the-sensuctl-cli)
+- [API Keys](#api-keys)
+   - [EXERCISE 3: Create an API Key](#exercise-3-create-an-api-key)
+- [Listing and Viewing Resources](#listing-and-viewing-resources)
+  - [EXERCISE 4: List Sensu Resources](#exercise-4-list-sensu-resources)
+- [Inventory Management](#inventory-management)
+  - [EXERCISE 5: List an Inventory of Nodes](#exercise-5-list-an-inventory-of-nodes)
+- [Monitoring as Code](#monitoring-as-code)
+  - [EXERCISE 6: Exporting and Updating Resource Configurations](#exercise-6-exporting-and-updating-resource-configurations)
 - [Learn more](#learn-more)
 - [Next steps](#next-steps)
 
-## Web App
+## Goals
 
-The Sensu backend process includes a web app that provides a unified view of your observability data (events, entities, and silences) and monitoring configuration (checks, handlers, etc) with user-friendly tools to reduce alert fatigue.
+In this lesson we will introduce the `sensuctl` command-line tool.
+You will learn how to install and configure the tool, then practice performing some essential operations.
```

Comment:
I am a little flip-floppy on this one. I think I like using "CLI" when referring to Sensu the product and when describing `sensuctl`, but like using "tool" when referring to `sensuctl` the program.

That's because here CLI is not really a noun as much as it is an adjective in this usage. It is a "CLI tool/app" vs a "GUI tool/app" .. But it's confusing because CLI is actually a noun phrase "command-line interface" (interface is the noun), which itself uses TWO nouns combined as an adjective for "interface" ("command-line")... So when we say something is a CLI app, it's four nouns in a row, with the first three serving as adjectives. This is a terrible grammatical thing that is all too prevalent in tech. In the style-guides and tech-writing classes we used to teach at Twitter these were called "noun piles", and writers were cautioned to avoid them and were offered some strategies for how to undo them. 

Usually by leaving out some of the extra specifying words that don't realllly need to be there, or by adding some grammatical words into the mix to make the pile more parsable. In this case, we've avoided this by making an acronym of CLI, which helps somewhat, but also doesn't. 

"The command-line interface for Sensu is `sensuctl`". 
"`sensuctl` is a tool that provides a command-line interface to Sesnu".

Anyway -- this is not an easy thing to answer and that leads to a lot of inconsistent usage in the text, depending on which phrasing flows best for that particular sentence/phrase/title/etc.

Reply:
Leaving as-is for now, as I don't think there's an obviously better word choice here, and it's just as understandable as written.